### ISSUE TITLE
'METADES' object has no attribute 'meta_classifier_'

### ISSUE THREAD

**Issue by s0rel** [scikit-learn-contrib/DESlib#187]
Hello, thanks for your work.Today I use metades and pass sklearn.linear_model.LogisticRegression as meta_classifier.Unfortunately, it doesn't work and throw an exception "AttributeError: 'METADES' object has no attribute 'meta_classifier_'"Here is my code:
```
from sklearn.ensemble import RandomForestClassifier
pool_classifiers = RandomForestClassifier()
pool_classifiers.fit(X_train, y_train)
from deslib.des.meta_des import METADES
from sklearn.linear_model import LogisticRegression
meta_classifier = LogisticRegression()
clf = METADES(pool_classifiers, meta_classifier=meta_classifier) 
clf.fit(X_dsel, y_dsel)
```
my envirement:python 3.7.3 sklearn 0.22.2 deslib 0.3
Please help me

**Comment by Menelau** [comment]
@s0rel Hello,

This is a bug that was fixed in recent patches. In order to fix that you need to install the latest version of the library using the following command: 

pip install git+https://github.com/scikit-learn-contrib/DESlib -U

That will install the current development version which has several bug fixes since version version 0.3 release

**Comment by s0rel** [comment]
@Menelau Thanks for your help.
Can you give me an example how to customize meta_classifier since I don't find examples about it in document.When I write codes as below
```
from sklearn.ensemble import RandomForestClassifier
pool_classifiers = RandomForestClassifier()
pool_classifiers.fit(X_train, y_train)
from deslib.des.meta_des import METADES
from sklearn.linear_model import LogisticRegression
meta_classifier = LogisticRegression()
clf = METADES(pool_classifiers, meta_classifier=meta_classifier) 
clf.fit(X_dsel, y_dsel)
```
I found errror as below:
```
ValueError: The meta-classifier should output probability estimates
```
Thanks again for your kindness
